README.md

```diff
@@ -186,6 +208,20 @@ WKT2 and PROJJSON are equivalent to one another - more clients understand WKT2,
 structure, since they are both JSON. For now it's probably best to use both for maximum interoperability, but just using PROJJSON 
 is likely ok if you aren't worried about legacy client support.
 
+### Thumbnails
+
+For (unlocated) thumbnails and similar imagery, it is recommended to add `proj:shape` with `proj:epsg` set to `null`
```

Comment:
```suggestion
For (unlocated) thumbnails and similar imagery, it is recommended set `proj:epsg` to `null` and include `proj:shape`
```

Slight wordsmithing to make it clear that `proj:shape` and `proj:epsg` are not linked. To me, 

> add `proj:shape` with `proj:epsg`

scans like `proj:epsg` is a component of `proj:shape`.

Reply:
Good point, updated.